cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+journey.
+
+There _are_ 'newborn hacks' of folds that allow you to use OS diapers on smaller
+babies but these are greatly dependent on the baby's body and on the diaper
+brands.
+
+# Our Choices
+
+Our choices have evolved over time, not only as we have gained experience
+but also as little A has grown and had different requirements.
+
+## Diaper Sizes
+I chose to invest in a small stash of newborn cloth diapers because I wanted
+to start as soon as possible. I wanted to learn the systems and understand
+what we preferred to use _before_ buying a full stash of OS diapers that we
+might hate because they were the wrong type or materials.
+
+Unless you don't plan to start cloth diapering until baby is 3 months old or
+so, I think this is a good investment because it lets you learn the ropes and
+they hold their value extremely well (more so than OS) so can be sold on easily.
+
+
+## Diaper Types
+Now that we have 'dialed things in' a little more, there are three types of
+diapers we use (there are others, but these are the only 3 in our rotation):
+
+### 1. All-In-Ones (AIO)
+This is our 'favourite' kind of diaper because it requires the **_least_ time
+to put together** by _far_. For the majority you just have to push the 'tongue'
+(sewn in insert) back into the pocket and that's it.
+With a small number (in our stash it's the Motherease Wizard Unos and
+Bellz Bumz Reusabelles), they're ready to go straight off the line.
+
+Whilst we now find we have to reinforce the majority of these with an extra
+3 layer bamboo insert, this isn't the case with all of them.
+
+**The biggest issue with AIOs** is that they take a lot longer to dry.
+
+**The key thing to be aware of** is that not all AIOs are created equally
+in terms of absorption. It is _very important_ to understand what the material
+composition of these diapers is so that you know what you're dealing with.
+
+For example:
+[Modern Cloth Nappies AIOs](https://www.modernclothnappies.co.uk/collections/pearl-pocket-all-in-one-nappies/products/pearl-pocket-all-in-one-reusable-one-size-nappy)
+(my absolute favourite AIOs from our stash), have a super absorbent
+hemp cotton mix insert.
+<img width="533" alt="image" src="https://user-images.githubusercontent.com/4185328/170531761-98c62f2c-a3ac-4eab-b941-04940c289e2f.png">
+
+The [Bambino Miosolo AIOs](https://www.bambinomio.com/collections/miosolo/products/miosolo-diaper)
+sit at the complete opposite end of that scale, with a microfibre core which,
+despite the claims on the website of 'super absorbency', has _very little
+absorbency_ and _very good marketing_.
+
+I wouldn't risk using one of these without reinforcement at any time of day.
+Why do we still have them you ask? Because they have a _great_ fit and are
+very practical in that sense!
+
+<img width="492" alt="image" src="https://user-images.githubusercontent.com/4185328/170533205-f0cfd3c2-b9f6-49a2-8f9c-702ea9f271cf.png">
```

Comment:
The contrast on this image is very poor and the font size is _tiny_ ... 
![inserts](https://user-images.githubusercontent.com/4185328/170533205-f0cfd3c2-b9f6-49a2-8f9c-702ea9f271cf.png)
Perhaps consider increasing the width to **`900px`** as the dimensions of the `.png` are **`984 × 956`**
so constraining it to **`492px`** in the **`<img>`** tag is artificially making it smaller than it needs to be without decreasing the bandwidth consumed (**`363kb`** ...) also, it's unclear what the **`40%`** in the top-right corner is. 🤷 

```suggestion
<img width="984" alt="image" src="https://user-images.githubusercontent.com/4185328/170533205-f0cfd3c2-b9f6-49a2-8f9c-702ea9f271cf.png">
```